Hypercube/src/app.php

```diff
@@ -13,10 +15,18 @@
 $app->register(new ServiceControllerServiceProvider());
 $app['hypercube.controller'] = function () use ($config) {
     return new HypercubeController(
-        new TesseractService($config['executable'])
+        new CmdExecuteService(),
```

Comment:
We can inject a logger, but we don't have to. Its set to a default:
https://github.com/Islandora-CLAW/Crayfish-Commons/blob/master/src/CmdExecuteService.php#L24